### PR TITLE
feat: add completed files counter

### LINKED ISSUE #30: Add counter to show how many files have been completed
# Feature Request

Idea initiated in #21, basically add a counter like `2/10` in the bottom right of the page to show how many files have been completed and how many are still left, the counter should show only after you complete at least one file.

### PR DESCRIPTION
**Pull request by lucasbento**
# Summary

This PR fixes #30, it adds a counter to the bottom right of the page showing how many files have been completed and how many are left, the counter only shows when you actually complete a file and it has a confetti animation that is triggered when you completed all of them.

## Test Plan

1. Run the PR locally;
1. Open http://localhost:3000/?from=0.59.8&to=0.59.9;
1. Complete files.

## Checklist

- [x] I tested this thoroughly
- [x] I added the documentation in `README.md` (if needed)

---

counter

### CODE REVIEW

**Comment by lucasbento** [comment]
cc @KevinHu2014 :)

**Comment by KevinHu2014** [comment]
The confetti is awesome 🎉🎉🎉

**Comment by KevinHu2014** [comment]
Just find out that there's different style behavior on the Safari browser :(

**Safari**

> 5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.1 Safari/605.1.15

螢幕快照 2019-07-05 上午1 27 58

**Chrome**
螢幕快照 2019-07-05 上午1 28 11

**Comment by kelset** [comment]
@KevinHu2014 let's create a dedicated issue so that someone can then submit a PR :)

File: src/components/common/CompletedFilesCounter.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  bottom: 20px;
+  right: 20px;
+  background: #d5eafd;
+  padding: 10px;
+  border: 1px solid #1890ff;
+  border-radius: 20px;
+  color: #7dadda;
+  transform: ${({ completed }) =>
+    completed ? 'translateY(0px)' : 'translateY(70px)'};
+  width: fit-content;
```
Comment (KevinHu2014) [inline src/components/common/CompletedFilesCounter.js]: After digging through what went wrong, it seems that` fit-content` doesn't went well on Safari. 
While I tested by changing it to `initial ` would went fine both for the `width` and `height`.

**Comment by KevinHu2014** [comment]
@kelset  Sure, I'll open the issue and send the PR :)

**Pull request merged**